src/Esfa.Vacancy.Application/Commands/CreateApprenticeship/Validators/WageTypeValidator.cs

```diff
@@ -23,9 +53,28 @@ private void ConfigureWageTypeValidator()
                 RuleFor(request => request.MaxWage)
                     .Null()
                     .WithErrorCode(ErrorCodes.CreateApprenticeship.MaxWage);
+
+                RuleFor(request => request.WageTypeReason)
+                    .Null()
+                    .WithErrorCode(ErrorCodes.CreateApprenticeship.WageTypeReason);
+            });
+
+            When(request => request.WageType == WageType.Unwaged, () =>
```

Comment:
Could you not squeeze these into OR statements in one set of definitions?

Reply:
its the auto gen'd message

Reply:
Hold that thought for when we have all the wageunit functionality in as well. May or may not make sense then.